src/state/exploredCohort.ts

```diff
@@ -39,9 +39,6 @@ const fetchExploredCohort = createAsyncThunk<
   const stateCohort = state.exploredCohort.cohort
   let shouldRefreshData = true
   switch (context) {
-    case 'cohort':
-      shouldRefreshData = !stateCohort || Array.isArray(stateCohort) || stateCohort.id !== id
-      break
```

Comment:
will this be a problem for aphp ?

Reply:
I guess yes

Reply:
Do you wish to check app context for this ?

Reply:
either a context check or a //FIXME (arkhn) comment to explain why we want to get rid of this line